### ISSUE TITLE
The submit button in surveys is difficult to see on the mobile site

### ISSUE THREAD

**Issue by dbrazel** [kevinwli/Genes-for-Good#3]
![screenshot_2015-04-06-12-49-20](https://cloud.githubusercontent.com/assets/4397488/7027015/6823cdb0-dd0a-11e4-9feb-d0e53aeedf34.png)

Samsung Galaxy S3, Android 4.4.2, Firefox for Android 37.0.1

**Comment by abecasis** [comment]
The other thing this question illustrates is that perhaps we should review the length of text on options (so they are mobile friendly).

**Comment by kevinwli** [comment]
The background image should be visible for webkit enabled browsers, such as
chrome and safari.

**Comment by dbrazel** [comment]
Yes, I can confirm that the background image is visible once I install Chrome for Android.
